### PR TITLE
7.50.0 changelog

### PR DESCRIPTION
**Pull request by kacper-murzyn**
Update of CHANGELOG and CHANGELOG-DCA for Agent 7.50.0 release.

### CODE REVIEW

File: CHANGELOG-DCA.rst
Comment (chouetz) [inline CHANGELOG-DCA.rst:1]: Is it expected that there is no mention of 7.49.1 in this document?

**Review by chouetz — APPROVED** [review]

File: CHANGELOG-DCA.rst
Comment (kacper-murzyn) [reply CHANGELOG-DCA.rst:1]: yes, there were no changes for DCA in 7.49.1 so there is nothing there

File: CHANGELOG-DCA.rst [hunk 1]
```diff
@@ -2,6 +2,55 @@
...
+.. _Release Notes_7.50.0_New Features:
+
+New Features
+------------
+
+- Add language detection API handler to the cluster-agent.
+
+- Report `rate_limit_queries_remaining_min` telemetry from `external-metrics` server.
+
+- Added a new --force option to the `datadog-cluster-agent clusterchecks rebalance` command that allows you to force clustercheck rebalancing with utilization.
```
Comment (hestonhoffman) [inline CHANGELOG-DCA.rst:19]:
```suggestion
- Added a new `--force` option to the `datadog-cluster-agent clusterchecks rebalance` command that allows you to force clustercheck rebalancing with utilization.
```

**Review by hestonhoffman — APPROVED** [review]
Tiny edit. LGTM

**Pull request merged**